spec/features/clusters/new_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe "cluster create", type: :feature do
+  describe "create page" do
+    it "curated cluster creation" do
+      user = create(:user)
+
+      visit "/users/#{user.username}/curated_clusters"
+      click_link "New Curated Cluster"
+      expect(page).to have_current_path("/users/#{user.username}/curated_clusters/new", ignore_query: true)
```

Comment:
does the "ignore_query" mean ignore everything after a ?

Reply:
only_path is deprecated and the new syntax is ignore_query